machAeroTutorials/airfoilopt_mesh.rst

```diff
@@ -53,7 +53,9 @@ General Options
 ``unattachedEdgesAreSymmetry``
     Tells pyHyp to automatically apply symmetry boundary conditions to any unattached edges (those that do not interface with another block).
 ``outerFaceBC``
-    Tells pyHyp to which boundary condition to apply to the outermost face of the extruded mesh.
+    Tells pyHyp to which boundary condition to apply to the outermost face of the extruded mesh (Note that we do not set the inlet or outlet boundaries seperately because it is automatically handled in adflow by the free stream flow direction).
+``BC``
+    Tells pyHyp to take the k=1 layer (surface of the airfoil) to be non-slip wall. Since it is a 2D problem, the jlow and jhigh  are set to be symmetry boundary conditions on both sides of the domain.
```

Comment:
I would swap the sentence order, because we just set the symmetry here. Then you add that by default the `k=1` (or `kLow`?) is set by default to be a non-slip wall.